src/Lut.ts

```diff
@@ -61,10 +61,79 @@ function remapDomainForCP(
 export const LUT_ENTRIES = 256;
 export const LUT_ARRAY_LENGTH = LUT_ENTRIES * 4;
 
+// @param {ControlPoint[]} controlPoints - array of {x:number 0..255, opacity:number 0..1, color:array of 3 numbers 0..255}
+// @return {Uint8Array} array of length len*4 representing the rgba values of the gradient
+function arrayFromControlPoints(controlPoints: ControlPoint[]): Uint8Array {
+  // current assumption is that control point X values are in the range 0-255
+  // and they will be used directly as indices into the LUT.
+  // therefore the lut must have 256 entries.  Anything else and we have to remap the control points.
```

Comment:
does this mean that if control points are remapped outside 0..255, as 3dvv does occasionally, this might not work?

Reply:
I think what I meant here is remapping them into something that can be used as a texture coordinate.  Because ideally the control point x value would be a raw intensity.

Reply:
Also what if we want more values in the lut, say 512 instead of 256. (maybe for reasons of better color interpolation or something)...